DirectEmployers/MyJobs
# [PD-3579] return correct report categories for site-traffic

## What's this PR do?
Fix a 500 error when report categories are not populated correctly.

## Screenshots (if appropriate) - 
error on the front end:

![image](https://user-images.githubusercontent.com/27963853/29941021-0161bcc8-8e5f-11e7-8162-81b784f061db.png)

error on the backend:

![image](https://user-images.githubusercontent.com/27963853/29941066-293f247e-8e5f-11e7-94e2-883da5b00685.png)

Request (`report` should have been `site-traffic.apply_clicks.apply_clicks`, indicating something wrong with report category data received by front end in previous call):
```
{
  "date_start":"05/04/2017 00:00:00",
  "date_end":"08/31/2017 15:13:18",
  "active_filters":[],
  "report":"site-traffic.user-agent.aggregated",
  "reporting_type":"site-traffic",
  "timeline":false
}
```

## How should this be manually tested?
Create a `new-site-traffic` switch, set it to active/not active and everything should still work.

## Any background context you want to provide?
Unable to reproduce what's happening in QC locally (possibly due to more frequent reloading), but this should fix the issue.
